steps/Asset.steps.ts

```diff
@@ -14,10 +14,11 @@ Then('Techsupport Employee Creates Category',async () => {
  await asset.categorytext()
```

Comment:
Add login using before block

Reply:
Added gitignore file and Added login using before block